src/main.cpp

```diff
@@ -20,10 +20,13 @@
 #include <QApplication>
 #include <QDebug>
 #include <QQmlApplicationEngine>
+#include <QClipboard>
 #include <QtQml>
+#include <QQmlContext>
```

Comment:
Include what you use:

```suggestion
#include <QtQml>
```

Reply:
Deleted